sqlglot/dialects/snowflake.py

```diff
@@ -887,6 +898,8 @@ class Generator(generator.Generator):
             ),
             exp.BitwiseXor: rename_func("BITXOR"),
             exp.BitwiseOr: rename_func("BITOR"),
+            exp.BitwiseLeftShift: lambda self, e: self.func("BITSHIFTLEFT", e.this, e.expression),
+            exp.BitwiseRightShift: lambda self, e: self.func("BITSHIFTRIGHT", e.this, e.expression),
```

Comment:
`self.func` should be used when we want to control which args are generated; For our case here, both expressions only have `this` and `expression` so we can simplify this into:

```
            ...
            exp.BitwiseLeftShift: rename_func("BITSHIFTLEFT"),
            exp.BitwiseRightShift: rename_func("BITSHIFTRIGHT"),
            ...
```

Reply:
I got overwhelmed by the `self.func`. I should have kept it simple. I've made the changes, please review it in your free time.